frontend/device/android/powerd.lua

```diff
@@ -75,6 +75,7 @@ function AndroidPowerD:turnOnFrontlightHW()
     if self:isFrontlightOn() and self:isFrontlightOnHW() then
         return
     end
+    android.enableFrontlightSwitch()
```

Comment:
On non-Tolinos this just a no-op, right?

Reply:
Yes, a method in `GenericController.kt` returning true.